pom.xml

```diff
@@ -129,8 +129,8 @@
     <build.path>build</build.path>
     <copycat.version>1.2.15</copycat.version>
     <glusterfs-hadoop.version>2.3.13</glusterfs-hadoop.version>
-    <grpc.version>1.31.0</grpc.version>
-    <netty.version>4.1.48.Final</netty.version>
+    <grpc.version>1.28.1</grpc.version>
+    <netty.version>4.1.51.Final</netty.version>
```

Comment:
Do these versions match the gRPC versions?

Reply:
https://github.com/grpc/grpc-java/blob/v1.28.1/repositories.bzl#L27
If we use gRPC 1.28.1 we need to use netty 4.1.45.Final

https://github.com/grpc/grpc-java/blob/v1.28.1/repositories.bzl#L27-L38

Reply:
nope. is that going to be a problem?  and this was done on purpose to avoid the GRPC regression but to address netty CVE.

Reply:
because we have some parts of our codebase that touch netty and gRPC relies heavily on netty, it can lead to runtime errors if the versions that we declare mismatch from gRPC

Reply:
it's not mandatory. We should be fine as soon as tests pass.